config-service/src/integrationTest/java/org/hypertrace/config/service/ConfigServiceIntegrationTest.java

```diff
@@ -162,14 +162,20 @@ public void testDeleteConfig() {
     upsertConfig(RESOURCE_NAME, RESOURCE_NAMESPACE, TENANT_1, Optional.of(CONTEXT_1), config2);
 
     // delete config with context
-    deleteConfig(RESOURCE_NAME, RESOURCE_NAMESPACE, TENANT_1, CONTEXT_1);
+    ContextSpecificConfig deletedConfig =
```

Comment:
please also update `MockGenericConfigService`

Reply:
Oh, forgot about it initially! Updated it now.